Extension/src/LanguageServer/extension.ts

```diff
@@ -221,51 +221,55 @@ function onInterval(): void {
 async function installVsix(vsixLocation: string, updateChannel: string): Promise<void> {
     // Get the path to the VSCode command -- replace logic later when VSCode allows calling of
     // workbench.extensions.action.installVSIX from TypeScript w/o instead popping up a file dialog
-    const platformInfo: PlatformInformation = await PlatformInformation.GetPlatformInformation();
-    const vsCodeScriptPath: string = function(platformInfo): string {
-        if (platformInfo.platform === 'win32') {
-            const vsCodeProcessPath: string = path.dirname(process.execPath);
-            const scriptName: string = path.basename(process.execPath) + '.cmd';
-            return path.join(vsCodeProcessPath, 'bin', scriptName);
-        } else {
-            const vsCodeProcessPath: string = path.basename(process.execPath);
-            const stdout: Buffer = execSync("which " + vsCodeProcessPath);
-            return stdout.toString().trim();
+    return PlatformInformation.GetPlatformInformation().then((platformInfo) => {
```

Comment:
I'm not sure why you removed `await` here.

Reply:
Good point! I'll add that back; it was more clear the other way